kytos/core/exceptions.py

```diff
@@ -77,6 +77,10 @@ def __str__(self):
         return msg
 
 
+class KytosResizingAvailableTagError(Exception):
+    """Exception raised when available_tag cannot be resized"""
```

Comment:
It'd be probably worth also introducing a new exception if the tag_type isn't supported too (I opened a discussion about it on your topology PR)

Reply:
Added `KytosTagtypeNotSupported`

Reply:
Great